### ISSUE TITLE
Cannot run multiple instances in parallel 

### ISSUE THREAD

**Issue by vmeurisse** [laurentj/slimerjs#18]
Hello,

When running two instances of SlimerJS in parallel, you get an error message:

> SlimerJS is already running, but is not responding. To open a new window, you must first 
> close the existing SlimerJS process, or restart your system.

PhantomJS handle this correctly.

Regards

**Comment by jamesryanbell** [comment]
This seems to be an issue for on Windows 7 x64. A new profile should be created on each instance if I understand the documentation properly but I am receiving an popup that says:

"SlimerJS is already running but is not responding. To open a new window, you must first close the existing SlimerJS proecss, or restart your system".

This works fine on Mac (Yosemite 10.10.1 (14B25)) so just seems to be a Windows issue.

**Comment by laurentj** [comment]
@jamesryanbell  Please create a new issue since this one is closed long time ago....

**Comment by jamesryanbell** [comment]
No problem I have created a new issue for this.

Thanks,
James
